recipes/libsvtav1/all/conanfile.py

```diff
@@ -43,6 +43,10 @@ def build_requirements(self):
         if self.settings.arch in ("x86", "x86_64"):
             self.tool_requires("nasm/2.15.05")
```

Comment:
Can you change cmake to 3.25.2 and have a check for `settings.os == "Linux".

So instead of:
```python
 if Version(self.version) >= "1.3.0":
            # at least CMake 3.16
            self.tool_requires("cmake/3.25.0")
```
have
```python
 if Version(self.version) >= "1.3.0" and self.settings.os == "Linux":
            # at least CMake 3.16
            self.tool_requires("cmake/3.25.2")
```

That might fix the conan v2 problem.

Reply:
V2 is missing packages, I would revert this :)